docs/configuration/logging.md

```diff
@@ -1,44 +1,19 @@
 # Logging
 
-This page contains two different sections with different targets:
+There are different solutions for you to aggregate logs in your cluster for day to day operation.
 
-- Aggregate logs for day to day opperation
-- Collect logs [for support](#collect-logs-for-support)
+Overall, all solutions follow the same model: you have an **agent** on each node to collect and forward the logs, an **aggregator** to persist the logs, and a **UI** that allows you to analyze the logs.
 
-## Aggregation of logs
+One of the simplest stacks that you could use is:
 
-There are a lot of different solutions for you to have logs in your
-cluster. Overall all of them follow the same model, you have an **agent**
-on each node, an **aggregator** to persist the logs and a **UI** to analyse the logs. 
+-   [Fluent Bit](https://github.com/helm/charts/tree/master/stable/fluent-bit) as the agent
+-   [Elasticsearch + Kibana](https://github.com/elastic/helm-charts/tree/master/elasticsearch) as the aggregator and UI
 
-One of the simplest stacks ou there is to use:
-
-1. [FluentBit](https://github.com/helm/charts/tree/master/stable/fluent-bit) as the agent
-2. [ElasticSearch + Kibana](https://github.com/elastic/helm-charts/tree/master/elasticsearch) for Aggregation and UI
-
-### Quick Installation
+To use the suggested stack, run the following commands to install the Fluent Bit and Elasticsearch charts on your cluster:
 
 ```bash
 helm repo add elastic https://helm.elastic.co
 helm install --name elasticsearch elastic/elasticsearch --namespace logging
 helm install --name fluent-bit stable/fluent-bit --namespace logging \
```

Comment:
The things here to setup logs are the one in the other page.
I do not even understand the use case for one of our clients to have elasticsearch

Reply:
Lets delete this page then @pauloribeiro-codacy if a customer asks for it we add it back

Reply:
Done!